### PR TITLE
Update configure-endpoints-script.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the important note with more information as requested by the user.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7087

File: microsoft-365/security/defender-endpoint/configure-endpoints-script.md [hunk 1]
```diff
@@ -33,8 +33,11 @@ You can also manually onboard individual devices to Defender for Endpoint. You m
 
 > [!IMPORTANT]
 > This script has been optimized for use on up to 10 devices.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-script.md]:
```suggestion
> This script has been optimized for use on up to ten devices.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-script.md [hunk 2]
```diff
@@ -33,8 +33,11 @@ You can also manually onboard individual devices to Defender for Endpoint. You m
...
->
-> To deploy at scale, use [other deployment options](configure-endpoints.md). For example, you can deploy an onboarding script to more than 10 devices in production with the script available in [Onboard Windows devices using Group Policy](configure-endpoints-gp.md).
+>Local scripting is a special on-board method for evaluating Microsoft Defender for Endpoint.
+>The data reporting frequency is set higher than the other on-board method when the on-board by local script is performed.
+>This setting is for evaluation purposes and is not normally used in production deployments.
+>For this reason, there are concerns about environmental impact, so please limit the number of deployments using local scripts to 10.
+>If you are deploying to a production environment as described above, use [other deployment options](configure-endpoints.md)like  Group Policy or Microsoft Endpoint Configuration Manager.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-script.md]:
```suggestion
> Local scripting is a special onboarding method for evaluating Microsoft Defender for Endpoint.
> The data reporting frequency is set higher than with other onboarding methods when onboarding using a local script.
> This setting is for evaluation purposes and is not normally used in production deployments. For this reason, there are concerns about environmental impact, so we recommend limiting the number of deployments using local scripts to ten.
> If you are deploying to a production environment as previously described, use [other deployment options](configure-endpoints.md) like  Group Policy or Microsoft Endpoint Configuration Manager.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Pleas merge this PR. Thanks

**Pull request merged**